.misc/travis_build_and_test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+if [[ ${UNSUPPORTED} == "false" || ${TRAVIS_BRANCH} == "master" ]]; then
+    py.test
```

Comment:
please ensure your script complies with https://github.com/coala/coala/pull/5243 ; ping @sangamcse

Reply:
@Asalle, re-push after changing this to 2-space indent. 😃